kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -245,6 +240,22 @@ window.ExerciseView = Backbone.View.extend({
 
             } else if (framework == "perseus") {
 
+                var item_index;
+
+                var assessment_items = self.data_model.get("all_assessment_items") || [{id: ""}];
+
+                if (typeof attempts !== "undefined") {
+
+                    item_index = attempts % assessment_items.length;
+
+                } else {
+                    
+                    item_index = Math.floor(Math.random() * assessment_items.length);
```

Comment:
If the former, then it seems as if it starts with a random index, then switches to the 2nd item in the list after that?